Tests/AppTests/Common/Helpers/Controllers/Registration/ApproveHelper.swift

```diff
@@ -0,0 +1,61 @@
+import Vapor
+import Foundation
+@testable import App
+
+//swiftlint:disable superfluous_disable_command
+//swiftlint:disable force_try
+final class ApproveHelper {
+  
+  @discardableResult
+  static func store(places: [Place] = [], cities: [City] = []) throws -> [App.Event]? {
+    
+    var cityId = [Identifier]()
+    var placeId = [Identifier]()
```

Comment:
In general preferred to use `var cityId: [Identifier] = []` in swift, so would be good to fix. 
Plus why you have array but property called `cityId` ? it should be `cityIds`

Reply:
fixed